src/app/user-sign-up/login/login.component.scss

```diff
@@ -0,0 +1,76 @@
+@import '../../layout/_color.scss';
+
+/* Log in Form container */
+.form-container {
+    border-radius: 15px;
+    box-shadow: $offBlack 0px 5px 15px;
+    padding: 20px;
+    width: 400px;
+
+    /* Form group contains input, label tags, validation */
+    .form-group {
+        display: flex;
+        flex-direction: column;
+    
+        /* Email and password input */
+        input[type=password], [type=email] {
```

Comment:
each property should be in separate line:
input[type=password], 
[type=email] {
    style here
}